src/Confluent.Kafka/Impl/NativeMethods/NativeMethods.cs

```diff
@@ -564,6 +564,18 @@ internal static extern IntPtr rd_kafka_AdminOptions_set_match_consumer_group_sta
                          ConsumerGroupState[] states,
                          UIntPtr statesCnt);
 
+        [DllImport(DllName, CallingConvention = CallingConvention.Cdecl)]
+        internal static extern IntPtr rd_kafka_AdminOptions_set_match_consumer_group_types(
+                        IntPtr options,
+                        ConsumerGroupType[] groupTypes,
+                        UIntPtr groupTypesCnt);
+
+        [DllImport(DllName, CallingConvention = CallingConvention.Cdecl)]
+        internal static extern IntPtr rd_kafka_AdminOptions_set_match_consumer_group_types(
+                        IntPtr options,
+                        ConsumerGroupType[] groupTypes,
+                        UIntPtr groupTypesCnt);
+
```

Comment:
Why is it defined two times? This is causing CI failure? Same in other NativeMethods files

Reply:
Actually these were due to rebasing of the "squash & merge" commits and these did not raise any merge conflict as well. So after local build/test and format, the last rebase messed these up. Would take care for next time !